SourceryRuntime/Sources/Common/TemplateContext.swift

```diff
@@ -48,7 +48,7 @@ public final class TemplateContext: NSObject, SourceryModel, NSCoding, Diffable
         let fileParserResultCopy: FileParserResult? = nil
 //      fileParserResultCopy = try? NSKeyedUnarchiver.unarchiveTopLevelObjectWithData(NSKeyedArchiver.archivedData(withRootObject: parserResult)) as? FileParserResult
 
```

Comment:
I would like to put a comment here for future reference, that `SwiftSyntax` may misbehave when used from background threads (see a [relevant comment here](https://github.com/krzysztofzablocki/Sourcery/pull/1063#issuecomment-1121337077)), thus `false` is hardcoded.